auth/http.go

```diff
@@ -1361,6 +1361,15 @@ func httpHandlerResetPasswordFinish(central *ImqsCentral, w http.ResponseWriter,
 	}
 	token := r.http.Header.Get("X-ResetToken")
 	password := strings.TrimSpace(r.http.Header.Get("X-NewPassword"))
+
+	//Authenticate password against forbidden list in auth config
+	for _, i := range central.Config.ForbiddenPassword {
```

Comment:
rather rename the `i` to something else, like `fPwd` or `fPassword` or `forbiddenPassword`. `i` is usually for numbers.